tasks/beats-redhat.yml

```diff
@@ -1,33 +1,47 @@
 ---
-
 - name: Ensure libselinux-python on CentOS 6.x
-  yum: name=libselinux-python state=present update_cache=yes
-  when: ( ansible_distribution == "CentOS" ) and ( ansible_distribution_major_version == "6" )
+  yum:
+    name: libselinux-python
+    state: present
+    update_cache: yes
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == "6"
 
 - name: Redhat - add beats repository
-  template: src=beats.repo.j2 dest=/etc/yum.repos.d/beats.repo
-  when: use_repository
+  template:
+    src: beats.repo.j2
+    dest: /etc/yum.repos.d/beats.repo
+  when: use_repository | bool
 
 - name: RedHat - install yum-version-lock
-  yum: name=yum-plugin-versionlock state=present update_cache=yes
+  yum:
+    name: yum-plugin-versionlock
+    state: present
+    update_cache: yes
 
-- name: RedHat - unlock {{beat}} for install
-  shell: yum versionlock delete {{beat}} || true
+- name: "RedHat - unlock {{ beat }} for install"
+  shell: "yum versionlock delete {{ beat }} || true"  # noqa 303
   changed_when: False
 
-- name: RedHat - Ensure {{beat}} is installed
-  yum: name={{beat}}{% if beats_version is defined and beats_version != "" %}-{{ beats_version }}{% endif %} state=present update_cache=yes
-  when: use_repository
+- name: "RedHat - Ensure {{ beat }} is installed"
+  yum:
+    name: "{{ beat }}{{ beats_version | bool | ternary('-{{ beats_version }}', '') }}"
+    state: present
+    update_cache: yes
+  when: use_repository | bool
   register: beat_install
   notify: restart beat
 
-- name: RedHat - lock {{beat}} version
-  shell: yum versionlock add {{beat}}{% if beats_version is defined and beats_version != "" %}-{{ beats_version }}{% endif %}
-  when: version_lock
+- name: "RedHat - lock {{ beat }} version"
+  shell: "yum versionlock add {{ beat }}{{ beats_version | bool | ternary('={{ beats_version }}', '') }}"  # noqa 303
```

Comment:
what is `noqa 303` comment?

Reply:
this comment is for ansible-lint, to ignore the error 303 (https://docs.ansible.com/ansible-lint/rules/default_rules.html)